backend/tournesol/migrations/0053_entitypollrating_sum_trust_scores.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.9 on 2023-05-22 20:25
```

Comment:
Just in case this file has been generated after our last batch upgrade: let's not forget to run `pip install -r requirements.txt` to sync our local packages.

Reply:
Done